kernel/configs/x86_64_kata_kvm_4.19.x

```diff
@@ -2960,6 +2968,7 @@ CONFIG_TEXTSEARCH_KMP=y
 CONFIG_TEXTSEARCH_BM=y
 CONFIG_TEXTSEARCH_FSM=y
 CONFIG_INTERVAL_TREE=y
+CONFIG_RADIX_TREE_MULTIORDER=y
```

Comment:
Heh, this confused me. It no longer exists in the HEAD of the kernel, but is still in 4.19. Let's hope we have made it to the 'kernel fragments' by the time we next do a major kernel upgrade, and remember to drop it... (I think the frags scripts will pick it up as an error at that point as 'unused').

Reply:
The "Radix" data-structure was replaced with the "Xarray" data-structure / API in v4.20.

Reply:
Yep. Kata is shipping longerm right now, so is on 4.19 - when we move off that we'll have to pick that change up (and hopefully our [new use of the kernels `merge_config.sh` to build a config from the CONFIG fragments we know we need](https://github.com/kata-containers/packaging/pull/314)) will pick that change up for us.